timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/ExclusiveLock.java

```diff
@@ -114,7 +114,7 @@ public void enqueue(LockRequest request) {
             LockRequest existingRequest = queue.put(request.requestId, request);
             if (existingRequest != null) {
                 queue.put(request.requestId, existingRequest);
-                throw new LoggableIllegalStateException(
+                throw new SafeIllegalStateException(
```

Comment:
Drive-by. The exception is exactly that, probably pre-dates the logsafe utility.

Reply:
aww ❤️ be the change you want to see in the world!